dist/src/main/java/io/camunda/zeebe/shared/management/ClusterEndpoint.java

```diff
@@ -70,6 +70,8 @@
 @Component
 @RestControllerEndpoint(id = "cluster")
 public class ClusterEndpoint {
+  private static final OffsetDateTime MIN_PARSER_COMPLIANT_DATE =
+      OffsetDateTime.parse("0000-01-01T00:00:00Z");
```

Comment:
While it's 4-digit indeed, I had some concerns with using the year zero :D
However according to the [spec](https://datatracker.ietf.org/doc/html/rfc3339#section-5.6) it's fine 🤷 no range constraint as for months and days.

also the go parser seems fine with it
```
	dateString := "0000-01-01T00:00:00Z"
	fmt.Println(time.Parse(time.RFC3339, dateString))
```
prints:
```
0000-01-01 00:00:00 +0000 UTC <nil>
```